docs/design/cni-testing-initiative-upstream.md

```diff
@@ -219,31 +219,31 @@ the semantics of it.
 |           +------------------+       +-------------------+                Figure 2:          |
```

Comment:
there is a TODO to fix this.. you wanna remove that? or there is more fixing needed?